pocs/utils/rs232.py

```diff
@@ -77,11 +81,11 @@ def __init__(self,
         self.ser.bytesize = serial.EIGHTBITS
         self.ser.parity = serial.PARITY_NONE
         self.ser.stopbits = serial.STOPBITS_ONE
-        self.ser.timeout = 2.0  # TODO(jamessynge): Allow caller or config to set.
+        self.ser.timeout = timeout
+        self.ser.write_timeout = timeout
```

Comment:
write_timeout of 0 means non-blocking, which might not be quite what you want.

Reply:
That's what I've been using as a test and has been working great both for the mount and the arduinos. Was going to ask you specifically about this. Not sure what would otherwise be a good default.

Reply:
Given that our messages are short, infrequent and that we (generally) wait for a response, 0 is probably a fine value for write_timeout.